### PR TITLE
#2107 Add delete dialog component unit tests

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2107)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Mui/Dialog/__tests__/DeleteDialog.test.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { cleanup, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import * as generate from 'utils/generate';
+import DeleteDialog from '../DeleteDialog';
+import { renderWithTheme } from 'utils/testUtils';
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Dialog/__tests__/DeleteDialog.test.js]: Please use `renderWithProvider` here. We're about to remove this utility from the `testUtils` :)

Comment (DavidMi-1) [reply ohara-manager/client/src/components/common/Mui/Dialog/__tests__/DeleteDialog.test.js]: ok :)

**Comment by DavidMi-1** [comment]
@konekoya Done modify :)

**Comment by konekoya** [comment]
Thanks!

**Pull request merged**